medcat/utils/meta_cat/ml_utils.py

```diff
@@ -57,9 +60,9 @@ def create_batch_piped_data(data: List[Tuple[List[int], int, Optional[int]]],
         y = torch.tensor([x[2] for x in data[start_ind:end_ind]], dtype=torch.long).to(device)
 
     x = torch.tensor(x, dtype=torch.long).to(device)
-    cpos = torch.tensor(cpos, dtype=torch.long).to(device)
-
-    return x, cpos, y
+    # cpos = torch.tensor(cpos, dtype=torch.long).to(device)
+    attention_masks = (x != 0).type(torch.int)
+    return x, cpos, attention_masks, y
```

Comment:
need to update doc string, why is cpos data movement movement commented out?

Reply:
This is due to earlier, cpos contained a single index, now it contains dynamic number of indices for each data point. Due to this, it cannot be converted to a tensor